catboost/docs/en/_includes/work_src/reusage/cli-regular-settings.md

```diff
@@ -2913,7 +2913,7 @@ The value of the `{{ ctr-types__TargetBorderCount }}` component overrides this p
 
 **{{ cli__params-table__title__default }}**
 
-{{ parameters__ctr-targer-border-count__default }}
+{{ parameters__ctr-target-border-count__default }}
```

Comment:
please also fix the definition in `catboost/docs/en/presets.yaml`

Reply:
sorry, i don't see any related commits

Reply:
Sorry, forgot to push